include/avif/avif.h

```diff
@@ -350,9 +352,12 @@ enum
     AVIF_TRANSFER_CHARACTERISTICS_SRGB = 13,
     AVIF_TRANSFER_CHARACTERISTICS_BT2020_10BIT = 14,
     AVIF_TRANSFER_CHARACTERISTICS_BT2020_12BIT = 15,
-    AVIF_TRANSFER_CHARACTERISTICS_SMPTE2084 = 16, // Perceptual Quantizer; BT.2100 PQ
+    AVIF_TRANSFER_CHARACTERISTICS_PQ = 16, // Perceptual Quantizer (HDR)
+    AVIF_TRANSFER_CHARACTERISTICS_BT2100_PQ = 16,
```

Comment:
personal optinion but I don't think the BT2100_ aliases are really necessary... PQ and HLG are already in the name